install.sh

```diff
@@ -3977,7 +4030,12 @@ install_into_existing_repositories() {
 ############################################################
 install_hooks_into_repo() {
     TARGET_ROOT="$1"
-    TARGET="${TARGET_ROOT}/.git"
+    TARGET="${TARGET_ROOT}"
+    IS_BARE=$(cd "${TARGET_ROOT}" && git rev-parse --is-bare-repository)
+
+    if [ "${IS_BARE}" = "false" ]; then
```

Comment:
Maybe personal preference, but this would read nicer for me:
```
if [ "${IS_BARE}" = "true" ]; then
    TARGET="${TARGET_ROOT}"
else
    TARGET="${TARGET_ROOT}/.git"
fi
```